src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp

```diff
@@ -347,16 +373,14 @@ void AttitudeEstimatorQ::task_main()
 			continue;
 		}
 
-		update_parameters(false);
-
 		// Update sensors
-		sensor_combined_s sensors;
+		sensor_combined_s sensors = _sub_sensor.get();
 
 		int best_gyro = 0;
 		int best_accel = 0;
 		int best_mag = 0;
 
-		if (!orb_copy(ORB_ID(sensor_combined), _sensors_sub, &sensors)) {
+		if (sensorsUpdated) {
```

Comment:
As far as I know, the usual way when polling would be to do:

```
if (fds[0].revents & POLLIN) {
```

instead of ```orb_check()```.